mlir-sycl/lib/Transforms/Inliner.cpp

```diff
@@ -52,6 +59,20 @@ static FunctionOpInterface getCalledFunction(const CallOpInterface &Call) {
   return nullptr;
 }
 
+static Optional<NamedAttribute> getPassThroughAttrs(CallOpInterface Call) {
+  constexpr StringLiteral PassThroughAttrName{"passthrough"};
+  FunctionOpInterface Callee = getCalledFunction(Call);
+  NamedAttrList FnAttrs(Callee->getAttrDictionary());
+  return FnAttrs.getNamed(PassThroughAttrName);
+}
+
+static bool hasAttribute(ArrayAttr Attrs, StringRef Name) {
+  return std::any_of(Attrs.begin(), Attrs.end(), [Name](Attribute Attr) {
```

Comment:
```suggestion
  return llvm::any_of(Attrs, [Name](Attribute Attr) {
```

Reply:
I'd rather use `std` (see above). I can change if we all agree, though.

Reply:
I prefer `llvm::any_of`, but I don't feel strongly about it.

Reply:
Reverted!